pillar/availability_zone.sls

```diff
@@ -0,0 +1,3 @@
+availability_zone:
+  region: 'region1'
+  zone: 'zone1'
```

Comment:
I can move them, no big deal.

However, the `{% if node_labels %}` isn't necessary (that's why I didn't put it there to begin with!)